std/net/curl.d

```diff
@@ -313,7 +313,7 @@ version(StdUnittest)
 version(StdDdoc) import std.stdio;
 
 // Default data timeout for Protocols
-enum _defaultDataTimeout = dur!"minutes"(2);
+private enum _defaultDataTimeout = dur!"minutes"(2);
```

Comment:
Undocumented public symbol, it's not too late for this one. It has been only added recently: https://github.com/dlang/phobos/commit/91eaa508c4